src/emulator/auth/state.ts

```diff
@@ -29,20 +29,30 @@ export class ProjectState {
   private oobs: Map<string, OobRecord> = new Map();
   private verificationCodes: Map<string, PhoneVerificationRecord> = new Map();
   private temporaryProofs: Map<string, TemporaryProofRecord> = new Map();
-  public oneAccountPerEmail = true;
-  private authCloudFunction: AuthCloudFunction;
-  public usageMode: UsageMode = UsageMode.DEFAULT;
+  protected _oneAccountPerEmail = true;
+  protected _authCloudFunction: AuthCloudFunction | undefined;
```

Comment:
nit: This should be `protected readonly` and should be passed in through a constructor argument. (Feel free to use the shorthand notation in constructor)